### DISCUSSION TITLE
Bypass "Made for kids" restrictions

### DISCUSSION THREAD

**Discussion by RobinRMC** [ReVanced/revanced-discussions#125] [Question]
The "measures" YouTube take to protect children are absurd, like not being able to add a video that is marked as "Made for kids" to a playlist.

I don't know if this is possible, but it would be useful if these restrictions could be bypassed somehow.

**Comment by oSumAtrIX** [comment]
Vanced has this feature already, so we will as well

**Comment by RobinRMC** [comment]
> Vanced has this feature already, so we will as well

Oh, I didn't know that.

**Comment by RobinRMC** [comment]
I checked it and Vanced doesn't have this feature.

**Comment by oSumAtrIX** [comment]
Do note, that the API does not restrict adding videos marked as "Made for kids" to playlists.  Although if we add a patch that enables adding the video anyways, the server could detect that. Not sure if that's safe to use

**Comment by Fooftilly** [comment]
You could already add those videos in the 3 dots menu in watch history on normal Youtube. I don't think that server would know if you have added from the watch history or from the video.

**Comment by oSumAtrIX** [comment]
fair enough

**Comment by RobinRMC** [comment]
> You could already add those videos in the 3 dots menu in watch history on normal Youtube. I don't think that server would know if you have added from the watch history or from the video.

I just wanted to say that. lol

**Comment by upintheairsheep** [comment]
I don't understand how miniplayer collects data
